releases/net471/dotnet471-changes.md

```diff
@@ -45,6 +41,7 @@ You can learn about the changes made in the .NET Framework 4.7.1, as of build 25
 * Added an attribute System.Runtime.CompilerServices.IsRefLikeAttribute [429642, mscorlib.dll, Feature]
 * Enabled interop between StorageFile / StorageFolder and .NET types for filesystem access [433898, mscorlib.dll, Feature]
 * System.Runtime.CompilerServices.RuntimeFeature.IsSupported allows to check whether the runtime supports displaying source line information for stack frames in assemblies built with portable symbols (PDBs). This is common for libraries that are built targeting .Net standard. [436315, mscorlib.dll, Feature]
+* During the .NET Standard 2.0 work a type was missed because of a mismatch between the contracts we have in full .NET and the ones we shipped as part of .NET Standard 1.*. [468256, System.Net.NetworkInformation.dll, Bug]
```

Comment:
comma after "work"
in full .NET --> in the .NET Framework

Reply:
Will fix